packages/amplication-client/src/Project/WorkspaceSelectorNew.scss

```diff
@@ -0,0 +1,87 @@
+@import "../style/index.scss";
+
+@mixin workspaceItem {
+  @include sectionTitle;
+  margin-bottom: 0;
+  padding: var(--default-spacing-small);
+  border-radius: var(--default-border-radius);
+  cursor: pointer;
+  white-space: nowrap;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+
+  .circle-badge {
+    margin-left: 0;
+    margin-right: var(--default-spacing);
+  }
+
+  &__name {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__plan {
+    @include body2;
+    color: var(--black60);
+    font-size: 10px;
+    line-height: 15px;
```

Comment:
use var or mixin

Reply:
@yuval-hazaz 
I am not sure how to use variables here.
I searched in the project for references, and I saw that that's how we write font size and line height when it is not a part of a mixin. AND none of the mixins we have is precisely what I need.

And another point - these properties and the above from your other comment is copy&paste from our current WorkspaceSelector CSS file